easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart

```diff
@@ -118,105 +118,113 @@ class NFTsListTile extends StatelessWidget {
+                              shrinkWrap: true,
+                              itemBuilder: (context, index) {
+                                return Container(
+                                  width: 30.0.w,
+                                  height: 9.0.h,
                                   color: EaselAppTheme.kLightGrey03,
-                                ),
-                              ),
-                              SizedBox(
-                                width: 10.w,
-                              ),
-                              Expanded(
-                                child: ListView.separated(
-                                  shrinkWrap: true,
-                                  itemBuilder: (context, index) {
-                                    return Container(
-                                      width: 30.0.w,
-                                      height: 9.0.h,
-                                      color: EaselAppTheme.kLightGrey03,
-                                    );
-                                  },
-                                  separatorBuilder: (context, index) {
-                                    return SizedBox(
-                                      height: 8.0.h,
-                                    );
-                                  },
-                                  itemCount: 3,
-                                ),
-                              ),
-                              SizedBox(
-                                width: 10.w,
-                              ),
-                              InkWell(
-                                onTap: () {},
-                                child: Padding(
-                                  padding: EdgeInsets.all(4.0.w),
-                                  child: SvgPicture.asset(
-                                    SVGUtils.kSvgMoreOption,
-                                    color: EaselAppTheme.kLightGrey03,
-                                  ),
-                                ),
-                              )
-                            ],
+                                );
+                              },
+                              separatorBuilder: (context, index) {
+                                return SizedBox(
+                                  height: 8.0.h,
+                                );
+                              },
+                              itemCount: 3,
+                            ),
                           ),
-                        ),
-                      );
-                    },
-                    errorWidget: (context, _, __) {
-                      return const IgnorePointer(child: SizedBox());
-                    },
-                  ),
-                ),
-              ) else const IgnorePointer(
-                child: SizedBox(),
+                          SizedBox(
+                            width: 10.w,
+                          ),
+                          InkWell(
```

Comment:
Why we have something that is not working?

Reply:
this is the place holder of the image, i guess that's why there is no feature implementation. 

<img width="768" alt="image" src="https://user-images.githubusercontent.com/115459890/201999079-2170357e-29ff-4840-91a5-e9095417be4a.png">

Reply:
If there is no use remove it.

Reply:
<img width="962" alt="image" src="https://user-images.githubusercontent.com/115459890/202125342-e828239f-65e3-460b-abd4-71e8ecd4fed3.png">


<img width="953" alt="image" src="https://user-images.githubusercontent.com/115459890/202125652-7df6fcf6-fb47-4870-8023-407b549e315a.png">


<img width="966" alt="image" src="https://user-images.githubusercontent.com/115459890/202126135-c03f544a-6759-4839-b8c3-e64b1a28ba85.png">

Reply:
@kjawadDeveloper2 kindly let me know what should I do.. should I change it all over the project?

Reply:
I will prefer shimmer.